tensorflow/stream_executor/rocm/rocm_blas.cc

```diff
@@ -1617,16 +1618,35 @@ bool ROCMBlas::DoBlasGemm(Stream *stream, blas::Transpose transa,
                       "precondition violation";
     }
   }
-  const Eigen::half alpha_half(alpha);
-  const Eigen::half beta_half(beta);
-  return DoBlasInternal(
+  bool hasXDLOPS = false;
+  auto status = GpuDriver::GetMFMASupport(hasXDLOPS);
```

Comment:
Explicit type preferred + is it actually checked?

Reply:
@cheshire pushed out a commit to make the type explicit and check the return status.

Please re-review.

thanks